ceph/ceph_admin/cephadm_ansible.py

```diff
@@ -39,7 +41,8 @@ def __init__(self, cluster):
 
     def install_cephadm_ansible(self):
         """Enable ansible rpm repos and install cephadm-ansible."""
-        if float(self.cluster._Ceph__rhcs_version) >= 6.0:
+        rhcs_version = re.match(r"(\d)\.(.?)(.*)", self.cluster._Ceph__rhcs_version)
+        if rhcs_version and float(rhcs_version.groups(0)[0]) >= 6.0:
```

Comment:
The logic works fine for rhcs_versions like '4.x', '5.3-rhel-8' etc. 

One suggestion here, can we do the grouping along with the re.match itself rather than in the if condition? Because, that will help us reuse the rhcs_version variable again without adding the .groups 

rhcs_version = re.match(r"(\d)\.(.?)(.*)", self.cluster._Ceph__rhcs_version).groups(0)[0]
if rhcs_version and float(rhcs_version) >= 6.0:

Reply:
Updated code.  Using `.groups` directly with `re.match` may throw error when unexpected string is passed.